server/@types/bapv.d.ts

```diff
@@ -3,6 +3,7 @@ import { PrisonerBasicInfoDto, VisitorBasicInfoDto } from '../data/orchestration
 export type Booker = {
   reference: string
   prisoners?: PrisonerBasicInfoDto[] // prisoners this booker can book for
+  visitors?: VisitorBasicInfoDto[] // visitors this booker can book for
```

Comment:
As discussed don't put this here; instead add `allVisitors` to `BookingJourneyData`. When the form is submitted we can have `selectedVisitors` in BookingJourneyData`